src/CarbonAware.WebApi/src/Filters/HttpResponseExceptionFilter.cs

```diff
@@ -46,12 +46,11 @@ public void OnException(ExceptionContext context)
             }
             var isVerboseApi = _options.CurrentValue.VerboseApi;
        
-            if (statusCode == (int)HttpStatusCode.InternalServerError && isVerboseApi == false)
+            if (statusCode == (int)HttpStatusCode.InternalServerError && !isVerboseApi)
             {
                  response = new HttpValidationProblemDetails() {
                                 Title = HttpStatusCode.InternalServerError.ToString(),
                                 Status = statusCode,
-                                Detail = context.Exception.Message
```

Comment:
I had to stare at the code a few times to see this 'one' place that makes the error less verbose :)